.github/workflows/deploy-xpr-testnet.yaml

```diff
@@ -1,14 +1,14 @@
-name: Build images for proton testnet environment
+name: Build images for xpr testnet environment
 
 on:
   push:
     tags:
-      - proton-testnet-v*
+      - xpr-testnet-v*
 
 jobs:
-  create-proton-image:
+  create-xpr-image:
     runs-on: ubuntu-latest
-    environment: proton-testnet
+    environment: xpr-testnet
```

Comment:
the environment must still be called `proton-testnet` in order to access the values stored as github secrets